R/pkg/R/functions.R

```diff
@@ -455,6 +455,19 @@ setMethod("acos",
             column(jc)
           })
 
+#' @details
```

Comment:
@zero323, should we add them into `generic.R` too?

Reply:
All trig functions are already generic, so we don't need that here.